### PR TITLE
Write the Docs: add doc for using configmap data in pods

### PR DESCRIPTION
**Pull request by jputrino**
This PR is part of the Write the Docs writing day effort. It adds a new doc for using ConfigMap data in pods and fixes some issues in the edited configmap task. (follow-on to PR #3759).

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3780)

### CODE REVIEW

**Comment by chenopis** [comment]
Sorry, there was error message during the preview build:

**Liquid Exception: Liquid syntax error (line 10): 'capture' tag was never closed in docs/tasks/configure-pod-container/configure-pod-configmap.md**

The issue might be putting `{% include task-tutorial-prereqs.md %}` as a bullet item. Try removing the asterisks on line 10.

File: docs/tasks/configure-pod-container/configure-pod-configmap.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+   kubectl exec -it redis redis-cli
+   127.0.0.1:6379> CONFIG GET maxmemory
+   1) "maxmemory"
+   2) "2097152"
+   127.0.0.1:6379> CONFIG GET maxmemory-policy
+   1) "maxmemory-policy"
+   2) "allkeys-lru"
+   ```
+
+{$ endcapture %}
```
Comment (jaharkes) [inline docs/tasks/configure-pod-container/configure-pod-configmap.md]: Looks like this should probable be `{% endcapture %}`

Comment (jputrino) [inline docs/tasks/configure-pod-container/configure-pod-configmap.md]: thanks!

**Comment by jputrino** [comment]
@chenopis - We chatted briefly about this on Tuesday, but I thought it'd be useful to get it on record here as well. The last section of this doc should probably be its own tutorial -- it deals with using configmaps to configure pods using redis as an example. LMK if you'd like me to open an issue for it.

**Comment by chenopis** [comment]
@jputrino Thanks for the reminder. I'll try and work on it later today. Yeah, go ahead and open an issue so that we can make sure it don't fall through the cracks. Thanks!

**Pull request merged**